cmake/generate_windows_import_lib.py

```
@@ -0,0 +1,62 @@
+    parser.add_argument("--dumpbin", required=True)
+    parser.add_argument("--dll", required=True)
+    parser.add_argument("--def", dest="def_file", required=True)
+    return parser.parse_args()
+
+
+def main() -> int:
+    args = parse_args()
+    dll_path = pathlib.Path(args.dll)
+    def_path = pathlib.Path(args.def_file)
+
+    proc = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)